ash/Cargo.toml

```diff
@@ -10,7 +10,8 @@ keywords = ["vulkan", "graphic"]
 documentation = "https://docs.rs/ash"
 
 [dependencies]
-shared_library = "0.1.5"
+#shared_library = "0.1.9"
```

Comment:
Will fix when 0.1.9 is out.

Reply:
What is the plan here? Because this library has been yanked from crates.io https://crates.io/crates/shared_library Will this be published again?

Reply:
Ah I see the "newer" versions have a lower version which messes up crates.io.